paddle/fluid/framework/ir/fuse_relu_depthwise_conv_pass.cc

```diff
@@ -25,9 +25,18 @@ namespace ir {
 
 std::unique_ptr<ir::Graph> FuseReluDepthwiseConvPass::ApplyImpl(
     std::unique_ptr<ir::Graph> graph) const {
-  VLOG(4) << "FuseReluDepthwiseConvPass::ApplyImpl";
+  graph = FuseReluDepthwiseConv(std::move(graph), true);
+  graph = FuseReluDepthwiseConv(std::move(graph), false);
```

Comment:
Because the fused graphs are different between training and inference, FuseReluDepthwiseConv runs twice for fusing both graphs in training and inference.

Reply:
They have different graph detect patterns.